src/static/stylus/base_template.styl

```diff
@@ -5,20 +5,54 @@
  */
 #bg
     z-index -1
-    background rgba(22, 12, 160, 0.9)
     position absolute
     top 0
     bottom 0
     left 0
     right 0
+    background rgba(19, 48, 70, 0.9)
 
 .ui.secondary.inverted.pointing.menu
     border transparent
 
 .jumbotron
-    min-height 700px
+    min-height 450px
     padding 1em 0
 
+#style_button
+    color #e2e2e2!important
+    background-color rgba(255, 255, 255, .15)!important
+    font-weight 100!important
+    box-shadow none!important
+
+#style_button:hover
+    color #3f3f3f!important
+    background-color rgba(255, 255, 255, .65)!important
+    box-shadow none!important
+    .icon
+        opacity 1 !important
+
+.ui.dropdown
+    position absolute!important
```

Comment:
can we get away from the `!important`'s? I'm not sure we can.. may have to research how selector precedence works.

Also, should have a space between the style and `!important`